### PR TITLE
SAFE [1/2] - Link to Safe page and individual Safe tx's

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

- Adds link to a project's Safe page from the project page (on the Safe badge icon):

https://user-images.githubusercontent.com/96150256/193720303-39c7e930-4301-4f1e-a6d9-9e1c08efec0e.mp4

- Adds icon to copy a link to individual Safe tx's -> scrolls to them, highlights in blue and expands collapse:

https://user-images.githubusercontent.com/96150256/193720751-fc1d1925-8baf-4bf3-921e-9bf0a344e8fe.mp4

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/v2v3/V2V3Project/ProjectSafeDashboard/ShareTxButton.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  if (transaction.isExecuted) {
+    linkToTx += `&tab=history`
+  }
+
+  const copyLinkToKeyboard = () => {
+    if (navigator) {
+      navigator.clipboard.writeText(linkToTx)
+      setCopied(true)
+      setTimeout(() => setCopied(false), 3000)
+    }
```
Comment (tomquirk) [inline src/components/v2v3/V2V3Project/ProjectSafeDashboard/ShareTxButton.tsx:36]: **suggestion**: could we instead use the existing `CopyTextButton` component?

File: src/components/v2v3/V2V3Project/ProjectSafeDashboard/ProjectSafeDashboard.tsx [hunk 2]
```diff
@@ -82,21 +111,29 @@ export function ProjectSafeDashboard() {
...
-              name={TAB_NAMES.history}
-              isSelected={selectedTab === SAFE_TX_HISTORY_KEY}
-              onClick={() => setSelectedTab(SAFE_TX_HISTORY_KEY)}
-            />
+            <Link href={`${projectSafeRoute}?tab=queued`}>
+              <a>
+                <Tab
+                  name={TAB_NAMES.queued}
+                  isSelected={selectedTab === SAFE_TX_QUEUED_KEY}
+                  onClick={() => setSelectedTab(SAFE_TX_QUEUED_KEY)}
```
Comment (tomquirk) [inline src/components/v2v3/V2V3Project/ProjectSafeDashboard/ProjectSafeDashboard.tsx:124]: **question** now that it's a link, can we remove the onClicks?

File: src/components/Project/ProjectHeader/GnosisSafeBadge.tsx [hunk 3]
```diff
@@ -1,21 +1,45 @@
...
+    <Link href={`${v2v3ProjectRoute({ projectId, handle })}/safe`}>
+      <a>
+        <img src={src} alt="Safe" width={15} height={15} />
+      </a>
+    </Link>
+  )
 }
 
 export function GnosisSafeBadge() {
+  const { handle } = useContext(V2V3ProjectContext)
```
Comment (tomquirk) [inline src/components/Project/ProjectHeader/GnosisSafeBadge.tsx:29]: Love this, great job

**Review by tomquirk — APPROVED** [review]
@johnnyd-eth I think this works really well! Nice job. I left a couple comments

**Review by tomquirk — CHANGES_REQUESTED** [review]
(sorry, didnt mean to approve)

File: src/components/v2v3/V2V3Project/ProjectSafeDashboard/ShareTxButton.tsx (hunk 1 above)
Comment (johnnyd-eth) [reply src/components/v2v3/V2V3Project/ProjectSafeDashboard/ShareTxButton.tsx:36]: Yep good call

File: src/components/v2v3/V2V3Project/ProjectSafeDashboard/ProjectSafeDashboard.tsx (hunk 2 above)
Comment (johnnyd-eth) [reply src/components/v2v3/V2V3Project/ProjectSafeDashboard/ProjectSafeDashboard.tsx:124]: Yep

**Comment by johnnyd-eth** [comment]
@tomquirk thanks for the comments. Moved the addresses to your two comments to this PR: https://github.com/jbx-protocol/juice-interface/pull/2126

**Review by tomquirk — APPROVED** [review]

**Pull request merged**